src/components/PsAccounts.vue

```diff
@@ -1,130 +1,145 @@
 <template>
   <div id="psaccounts">
-    <BaseAlert
-      variant="danger"
-      :show="hasError"
-      dismissible
-      @dismissed="hasError = false"
-    >
-      <p class="acc-text-sm acc-leading-6">
-        {{ $t('psaccounts.accountManager.errorInstallEnable') }}
-      </p>
-    </BaseAlert>
-
-    <BaseAlert
-      v-if="validContext.errors && validContext.errors.length"
-      :class="{'acc-mt-4': hasError}"
-      variant="danger"
-    >
-      <p class="acc-text-sm acc-leading-6">
-        &lt;PsAccounts&gt; integration: Given context is invalid:
-        {{ validContext.errors.join(';') }}
-      </p>
-    </BaseAlert>
+    <ContextValidatorAlert
+      v-if="errors.length"
+      :errors="errors"
+      data-testid="account-context-validator-alert"
+    />
     <template v-else>
-      <PsAccountComponentAlertDisplay
+      <ModuleDependenciesAlert
+        :ps-accounts-is-enabled="context.psAccountsIsEnabled"
+        :ps-accounts-enable-link="context.psAccountsEnableLink"
+        :ps-accounts-is-installed="context.psAccountsIsInstalled"
+        :ps-accounts-install-link="context.psAccountsInstallLink"
+        :ps-accounts-is-uptodate="context.psAccountsIsUptodate"
+        :ps-accounts-update-link="context.psAccountsUpdateLink"
```

Comment:
A property for ps_accounts state and another with object for all can be more readable, what do u think ?

Reply:
I agree but i think this object should be provided by the ps_accounts module

Reply:
Yes, the context it's not very smart but let's avoid creating BR in each project simultaneous